types/webpack/index.d.ts

```diff
@@ -646,6 +647,118 @@ declare namespace webpack {
             /** Generate records with relative paths to be able to move the context folder". */
             portableRecords?: boolean;
         }
+        interface DevServerOptions {
+            /** Provides the ability to execute custom middleware after all other middleware internally within the server. */
+            after?: () => void;
```

Comment:
This doesn’t look right. Based on the docs the function has a parameter.

https://webpack.js.org/configuration/dev-server/#devserver-after